homeassistant/components/weatherflow_cloud/strings.json

```diff
@@ -23,5 +23,73 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_service%]",
       "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "air_temperature": {
+        "name": "[%key:component::sensor::entity_component::temperature::name%]"
+      },
+      "air_density": {
+        "name": "Air density"
+      },
+      "barometric_pressure": {
+        "name": "Pressure barometric"
+      },
+      "sea_level_pressure": {
+        "name": "Pressure sea level"
+      },
+      "station_pressure": {
+        "name": "Pressure station"
+      },
+      "dew_point": {
+        "name": "Dew point"
+      },
+      "lightning_strike_count": {
+        "name": "Lightning count"
+      },
+      "lightning_strike_count_last_1hr": {
+        "name": "Lightning count last 1 hr"
+      },
+      "lightning_strike_count_last_3hr": {
+        "name": "Lightning count last 3 hr"
+      },
+      "lightning_strike_last_distance": {
+        "name": "Lightning last distance"
+      },
+      "lightning_strike_last_epoch": {
+        "name": "Lightning last"
```

Comment:
last what

Reply:
When the last strike was -> so it will say (in-app):

<img width="206" alt="image" src="https://github.com/home-assistant/core/assets/6491743/1fad869a-8ad7-4d96-9761-43e538bae0ad">

What do you suggest? I'm open to an idea:

Last Strike?

Reply:
Last lighting strike